### PR TITLE
feat: Sandpack POC

### PR DESCRIPTION
**Pull request by suevalov**
# Purpose of PR

An example of using Sandpack. 

**Benefits**

* Everything that codesandbox can do, but in our documentation.
* Each example can be opened as Codesandox
* Loads and compiles fast 

**Cons**

* Looks like there's no way to link a local copy of Forma. Basically the examples will be run with **the latest published** version of Forma.

### CODE REVIEW

**Comment by gui-santos** [comment]
> **Cons**
> 
> * Looks like there's no way to link a local copy of Forma. Basically the examples will be run with **the latest published** version of Forma.

Isn't this already a big problem?
It means that when we open a PR the preview website won't have the correct version and the examples may fail
previews of the website won't be reliable basically

File: packages/website/components/SandpackRenderer.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Sandpack } from '@codesandbox/sandpack-react';
+
+const indexFile = `import React, { StrictMode } from "react";
+import ReactDOM from "react-dom";
+import { GlobalStyles } from "@contentful/f36-components";
+import "./styles.css";
```
Comment (gui-santos) [inline packages/website/components/SandpackRenderer.tsx:7]: I think this file does not exist

**Review by gui-santos — CHANGES_REQUESTED** [review]
It looks cool, but the only Con issue worries me a lot

File: packages/website/components/SandpackRenderer.tsx (hunk 1 above)
Comment (suevalov) [reply packages/website/components/SandpackRenderer.tsx:7]: it exists when but not visible for edit

**Comment by suevalov** [comment]
@gui-santos yeah, I think it's a big problem so my suggestion would be not to proceed with the solution

**Review by bgutsol — COMMENTED** [review]
Looks great and works fine. Love it! 👏 
Curious if there will be any performance drawdowns if we have more than 10 examples on the same page, like the` Menu` component has

**Pull request closed without merging**